### PR TITLE
[14.0][ADD] sale_order_report_delivered

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Adds new sale order report with delivered quantity

It remains to solve the problem that does not show the taxes

### CODE REVIEW

**Comment by ChristianSantamaria** [comment]
imagen

Done, @dalonsod, can you test it?

File: sale_order_report_delivered/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            line_ids.append(valued_line)
+        return line_ids
+
+    def get_qty_delivered_subtotal(self):
+        price_subtotal = 0
+        for line in self.get_qty_delivered_lines():
+            price_subtotal += line.price_subtotal
+        return price_subtotal
+
+    def get_qty_delivered_taxes(self):
```
Comment (dalonsod) [inline sale_order_report_delivered/models/sale_order.py]: This code, is inspired/copied from which Odoo file? 😇
I would take a look at it

Comment (ChristianSantamaria) [reply sale_order_report_delivered/models/sale_order.py]: Its a copy of this: https://github.com/odoo/odoo/blob/ab0ae8dadf55baa258d733835acc3a12771932be/addons/sale/models/sale.py#L983
But with the temp lines

**Comment by ChristianSantamaria** [comment]
Warning: two last commit have errors, the stable version is 1cc2296f5d201b1ecb534773adc1190645e520d8

**Comment by dalonsod** [comment]
Current code may be fixed, but only tested with one line and one tax. @ChristianSantamaria could you test it with more data?

**Comment by ChristianSantamaria** [comment]
Localy tested multiple taxes and lines. LGTM 👍

**Comment by dalonsod** [comment]
👍 I miss proforma report (currently not used in customer environment, anyway), could you add it?

**Comment by dalonsod** [comment]
Final changes made:

- Restored "Deliv. Qty." as column header, now these two reports are visually different,
- Added myself as author.

Installed in CigTest 👌🏽

**Pull request merged**